modules/cnf-configuring-the-ptp-fast-event-publisher.adoc

```diff
@@ -66,6 +65,45 @@ If `storageType` is not set and the `transportHost` uses HTTP, the PTP daemons a
 $ oc apply -f ptp-operatorconfig.yaml
 ----
 
+. Create a `Service` resource for the events HTTP transport. For example:
```

Comment:
Sorry I didn't make it clear -  this service is for consumer only. This step should be removed from here. It is already documented in Step 3 of #cnf-migrating-from-amqp-to-http-transport_using-ptp